contracts/Makefile

```
@@ -0,0 +1,7 @@
+setup:
+	rustup target add wasm32-unknown-unknown
+	cargo fetch
+	cargo build --target wasm32-unknown-unknown
+
+test:
```

checkmake (0.3.2)

[warning] Required target "all" is missing from the Makefile.

(minphony)